src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -342,10 +326,6 @@ public static class Avx
         public static unsafe Vector128<double> MaskLoad(double* address, Vector128<ulong> mask) { throw null; }
         public static unsafe Vector256<float> MaskLoad(float* address, Vector256<uint> mask) { throw null; }
         public static unsafe Vector256<double> MaskLoad(double* address, Vector256<ulong> mask) { throw null; }
-        public static unsafe void MaskStore(float* address, Vector128<float> mask, Vector128<uint> source) { throw null; }
```

Comment:
@fiigii - we had a discussion about these methods right?  You wanted to change the type of the 3rd parameter. Are these not yet implemented?

Reply:
These methods are not implemented